ios/l2_interfaces/deleted_example_01.txt

```diff
@@ -0,0 +1,116 @@
+# Using Deleted
+
+# Before state:
+# -------------
+#
+# viosl2#show interfaces switchport
```

Comment:
IMHO, showing the running configuration for the interfaces should be enough here ( and below)

Reply:
@cidrblock have updated the before n after state e.g. to use `show running-config | section ^interface` instead of `show interfaces switchport`.